google_compute_engine_oslogin/utils/oslogin_utils_test.cc

```diff
@@ -426,7 +426,7 @@ TEST(ParseJsonSshKeyTest, ParseJsonToSshKeysFiltersMalformedExpiration) {
 
 TEST(ParseJsonAuthorizeSuccess, SuccessfullyAuthorized) {
   string response = "{\"success\": true}";
-  ASSERT_TRUE(ParseJsonToAuthorizeResponse(response));
+  ASSERT_TRUE(ParseJsonToSuccess(response));
```

Comment:
I would also add new unit test cases that cover the expected behavior of new utility methods you add